src/factory/src/state.rs

```diff
@@ -49,11 +52,35 @@ impl Default for State {
     }
 }
 
-pub fn get_token_bytecode() -> &'static [u8] {
-    include_bytes!("token.wasm")
+pub fn get_token_bytecode() -> Vec<u8> {
+    State::get().borrow().token_wasm.clone().unwrap_or_default()
 }
 
-ic_helpers::impl_factory_state_management!(State, get_token_bytecode());
+impl State {
+    pub fn stable_save(&self) {
+        ::ic_cdk::storage::stable_save((self,)).unwrap();
+    }
+
+    pub fn stable_restore() {
+        let (mut loaded,): (Self,) = ::ic_cdk::storage::stable_restore().unwrap();
+        let _ = loaded.token_wasm.take();
```

Comment:
Since we now allow the controller to change the token canister code at any time, do we really need to remove it after factory upgrade? If not, can't we use the default state management methods (from the `impl_factory_state_management`)?